experiment/scheduler.py

```diff
@@ -715,8 +715,14 @@ def render_startup_script_template(instance_name: str, fuzzer: str,
     """Render the startup script using the template and the parameters
     provided and return the result."""
     fuzzer_config = fuzzer_config_utils.get_by_variant_name(fuzzer)
+    local_experiment = experiment_utils.is_local_experiment()
+    if not local_experiment:
+        experiment_config['docker_registry'] = (
+            experiment_utils.get_base_docker_tag(
+                experiment_config['cloud_project']))
```

Comment:
I prefer not to make `docker_registry` as a required one for cloud running. @jonathanmetzman wdyt?
Do we need to make `docker_registry` also required option while it can be generated by `cloud_project`, where `cloud_project` is already required.

Reply:
Right, I already set that in the run_experiment.py.
Here is for unit test passing.
I will remove this and enhance the unit test environment.

Reply:
+1 to enhancing test!